hammett/stopwatch/collector.py

```diff
@@ -0,0 +1,84 @@
+"""The module contains a decorator that wraps handlers to track time."""
```

Comment:
`The module contains a decorator that wraps handlers to track their execution time.`